src/yolov7_reid/src/LabelStudioClient.py

```diff
@@ -25,12 +25,12 @@ def upload_file(file_path):
             file_id = None
             if file_ids is not None and len(file_ids) > 0:
                 file_id = file_ids[0]
-            if file_id is not None:
+            if file_id is not None and response.ok:
```

Comment:
Yep, this was a bug, since line 21 already has `if response.ok:` so it always `true` here